bigtable/connectionpool/configure_connection_pool_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connectionpool
+
+import (
+	"bytes"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestConnectionPool(t *testing.T) {
```

Comment:
```suggestion
func TestConfigureConnectionPool(t *testing.T) {
```

(Convention to name the test after the thing being tested.)